static/js/shared/util.js

```diff
@@ -20,7 +20,7 @@ const POPULATION = "StatisticalPopulation";
 const OBSERVATION = "Observation";
 const COMPARATIVE_OBSERVATION = "ComparativeObservation";
 
-const MAX_CARD_HEIGHT = 400;
+const MAX_CARD_HEIGHT = 450;
```

Comment:
Are we sure we want the charts to be so tall? The pages are already very long...

Reply:
this is what I see now: 
![image](https://user-images.githubusercontent.com/5951856/89685576-74496700-d8b1-11ea-9ba8-ed64358ab235.png)

Or, we can reduce the chart size?

Reply:
Oh got it. I thought we were making the chart itself taller. The ratio feels appropriate for the width, sounds good then.